data-structures-stacks/max-value.js

```diff
@@ -0,0 +1,13 @@
+export default function maxValue(stack) {
+  let biggy = stack.pop();
+  if (stack.peek() === undefined) {
+    return -Infinity;
```

Comment:
I'm not convinced this works. If there is exactly one item in the stack it returns `-Infinity` instead of that single value.

Oh, wow, the test doesn't test this case! I'm adding it right now!